cmd/logger.go

```diff
@@ -137,7 +108,32 @@ func getSource() string {
 	return fmt.Sprintf("[%s:%d:%s()]", filename, lineNum, funcName)
 }
 
-func logIf(level logrus.Level, source string, err error, msg string, data ...interface{}) {
+// trace method - to create stack trace
+func getTrace(level int) []string {
+	var trace []string
+	var pc uintptr
+	var file string
+	var lineNumber int
+	ok := true
```

Comment:
you could clear `ok` than initializing with `true` i.e. `var ok bool`

Reply:
Good point.
Actually we can initialize `pc`, `file`, `lineNumber`, and `ok` in the next line:
`pc, file, lineNumber, ok := runtime.Caller(level)`
without declaring them first. So, I removed the declarations completely.